app/views_redesigned/campaigns/index.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+json.array! @campaigns, partial: "campaigns/campaign", as: :campaign
```

Comment:
Are we missing the `app/views_redesigned/campaigns/campaign.json.jbuilder` partial?

Reply:
Ah good catch, I accidentally removed it from this PR. I’ll add that in and merge.